src/test/java/kitchenpos/application/menu/MenuServiceTest.java

```diff
@@ -0,0 +1,219 @@
+                menuDao,
+                menuGroupDao,
+                menuProductDao,
+                productDao
+        );
+    }
+
+    @DisplayName("메뉴 가격")
+    @Nested
+    class MenuPriceNestedTest {
+
+        @DisplayName("[SUCCESS] 등록할 신규 메뉴에 메뉴 상품이 없을 경우 메뉴의 가격은 0원이어야 등록할 수 있다.")
+        @Test
+        void success_create_menu_when_MenuProductsIsEmpty() {
+            // given
+            final MenuGroup savedMenuGroup = menuGroupDao.save(new MenuGroup("테스트용 메뉴 그룹명"));
+
+            final BigDecimal zeroPrice = BigDecimal.ZERO;
+            final Menu expected = new Menu(
+                    "테스트용 메뉴명",
+                    zeroPrice,
+                    savedMenuGroup.getId(),
+                    Collections.emptyList()
+            );
+
+            // when
+            final Menu actual = menuService.create(expected);
+
+            // then
+            assertSoftly(softly -> {
+                softly.assertThat(actual.getId()).isPositive();
+                softly.assertThat(actual.getName()).isEqualTo(expected.getName());
+                softly.assertThat(actual.getPrice()).isEqualByComparingTo(expected.getPrice());
+                softly.assertThat(actual.getMenuGroupId()).isEqualTo(expected.getMenuGroupId());
+                softly.assertThat(actual.getMenuProducts()).isEqualTo(expected.getMenuProducts());
+            });
+        }
+
+        @DisplayName("[EXCEPTION] 등록할 신규 메뉴에 가격이 null 일 경우 예외가 발생한다.")
+        @Test
+        void throwException_when_create_Menu_IfMenuPriceIsNull() {
+            // given
+            final MenuGroup savedMenuGroup = menuGroupDao.save(new MenuGroup("테스트용 메뉴 그룹명"));
+
+            // when
+            final Menu expected = new Menu(
+                    "테스트용 메뉴명",
+                    null,
+                    savedMenuGroup.getId(),
+                    Collections.emptyList()
+            );
+
+            // then
+            assertThatThrownBy(() -> menuService.create(expected))
+                    .isInstanceOf(IllegalArgumentException.class);
+        }
+
+        @DisplayName("[EXCEPTION] 등록할 신규 메뉴에 가격이 음수일 경우 예외가 발생한다.")
+        @ParameterizedTest
+        @ValueSource(strings = {"-1", "-10", "-100", "-1000000"})
+        void throwException_when_create_Menu_IfMenuPriceIsNegative(final String negativePriceValue) {
+            // given
+            final MenuGroup savedMenuGroup = menuGroupDao.save(new MenuGroup("테스트용 메뉴 그룹명"));
+
+            // when
+            final Menu expected = new Menu(
+                    "테스트용 메뉴명",
+                    new BigDecimal(negativePriceValue),
+                    savedMenuGroup.getId(),
+                    Collections.emptyList()
+            );
+
+            // then
+            assertThatThrownBy(() -> menuService.create(expected))
+                    .isInstanceOf(IllegalArgumentException.class);
+        }
+
+        @DisplayName("[EXCEPTION] 등록할 신규 메뉴에 메뉴 상품 목록들의 가격 합보다 메뉴 상품의 가격이 높을 경우 예외가 발생한다.")
```

Comment:
```suggestion
        @DisplayName("[EXCEPTION] 등록할 신규 메뉴에 메뉴 상품 목록들의 가격 합보다 메뉴의 가격이 높을 경우 예외가 발생한다.")
```

검증 대상이 메뉴의 가격이니 메뉴 가격이라고 적는게 더 적절해보여요😊